### ISSUE TITLE
Package.getActivationCommands is deprecated.

### ISSUE THREAD

**Issue by HadrienG** [AtomLinter/linter-shellcheck#12]
Use `activationCommands` instead of `activationEvents` in your package.json
Commands should be grouped by selector as follows:
```json
  "activationCommands": {
    "atom-workspace": ["foo:bar", "foo:baz"],
    "atom-text-editor": ["foo:quux"]
  }
```
```
Package.getActivationCommands (/Applications/Atom.app/Contents/Resources/app.asar/src/package.js:790:9)
Package.hasActivationCommands (/Applications/Atom.app/Contents/Resources/app.asar/src/package.js:715:20)
<unknown> (/Applications/Atom.app/Contents/Resources/app.asar/src/package.js:169:24)
Package.measure (/Applications/Atom.app/Contents/Resources/app.asar/src/package.js:147:15)
```

**Comment by marcos-mestre** [comment]
+1

**Comment by kevinsawicki** [comment]
Fixed by #15 and published in `0.0.8`
